luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/factories/KeyValueStoreDimensionFactorySpec.groovy

```diff
@@ -41,7 +46,7 @@ class KeyValueStoreDimensionFactorySpec extends Specification {
             testDimension.getCategory() == "a category for testing"
             testDimension.getDescription() == "a description for testing"
             testDimension.getStorageStrategy().getApiName() == "loaded"
-            testDimension.isAggregatable() == false
+            //testDimension.isAggregatable() == false
```

Comment:
Is this temporarily making the tests pass, due to inconsistencies caused by rebasing?

If that is the case you can try and run `lua config.lua` after `cd`ing to the `fili/luthier/config/src/config.lua` directory. In any given time the newly generated json should pass any tests or I did something wrong.